bin/helpers/index.js

```diff
@@ -0,0 +1,5 @@
+const { possibleLocations } = require('./possibleLocations');
```

Comment:
Looks like we need to bring some standard conventions for organizing files, can we create a `src` dir at the root and organize the helpers there.

Reply:
Sounds good to me, do you like this structure?

```
alacrity-themes
  src/
    helpers/
      possibleLocations.js
  bin/
    cli.js
```

Reply:
Yes